api/response/schema_query.py

```diff
@@ -1,11 +1,18 @@
 import graphene
+from graphene_sqlalchemy import (SQLAlchemyObjectType)
 from graphql import GraphQLError
 from api.response.schema import Response
 from api.room_resource.models import Resource
 from api.room.schema import Room
 from api.room.models import Room as RoomModel
 from helpers.auth.authentication import Auth
 from helpers.pagination.paginate import ListPaginate
+from api.response.models import Response as ResponseModel
+
+
+class Responses(SQLAlchemyObjectType):
+    class Meta:
+        model = ResponseModel
 
 
 class ResponseDetails(graphene.ObjectType):
```

Comment:
@nakatuddesuzan kindly add docstrings to these classes. this is good work.

Reply:
This has been resolved @dnuwa